torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6266,6 +6281,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            supports_out=False,
            sample_inputs_func=sample_movedim_moveaxis),
+    OpInfo('renorm',
+           dtypes=floating_types(),
+           dtypesIfCUDA=floating_types_and(torch.float16),
+           sample_inputs_func=sample_inputs_renorm,
+           skips=(
+               # AssertionError: Resized a non-empty tensor but did not warn about it.
```

Comment:
I guess this isn't simple to fix because renorm is still in TH?

Reply:
Yes.